lib/rules/require-ember-lifeline.js

```diff
@@ -6,6 +6,7 @@ const { getCaller, cleanCaller } = require('../utils/caller');
 const { getEmberImportBinding } = require('../utils/imports');
 const { collectObjectPatternBindings } = require('../utils/destructed-binding');
 
+const LINTABLE_FILE_PATTERNS = /components|routes|controllers|services/i;
```

Comment:
This should probably have `/` before and after the matching group:

```js
/\/(components|routes|controllers|services)\//i
```

To ensure it doesn't match some edge case like `helpers/lookup-services`.

Reply:
Good call. Will add.